l10n_it_partially_deductible_vat/__openerp__.py

```diff
@@ -24,27 +24,25 @@
 ##############################################################################
 
 {
-    "name" : "Italy - Partially Deductible VAT",
-    "version" : "0.1",
-    "depends" : ['account'],
-    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
+    "name": "Italy - Partially Deductible VAT",
+    "version": "0.1",
+    "depends": ['account'],
+    "author": "OpenERP Italian Community,Odoo Community Association (OCA),"
+              "SimplERP srl",
     "description": """
```

Comment:
Uhm, do I have to remove simplerp srl?
No problem

Sergio Corato
Product Manager
SimplERP srl
Il 31/lug/2015 14:40, "Davide Corio" <notifications@github.com> ha scritto:

> In l10n_it_partially_deductible_vat/__openerp__.py
> <https://github.com/OCA/l10n-italy/pull/127#discussion_r35969444>:
>
> > @@ -24,27 +24,25 @@
> >  ##############################################################################
> >
> >  {
> > -    "name" : "Italy - Partially Deductible VAT",
> > -    "version" : "0.1",
> > -    "depends" : ['account'],
> > -    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
> > +    "name": "Italy - Partially Deductible VAT",
> > +    "version": "0.1",
> > +    "depends": ['account'],
> > +    "author": "OpenERP Italian Community,Odoo Community Association (OCA),"
> > +              "SimplERP srl",
> >      "description": """
>
> @tafaRU <https://github.com/tafaRU> oh great, so that's the magic behind.
> thanks
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/l10n-italy/pull/127/files#r35969444>.
>